{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -308,16 +309,25 @@
 #
 # Custom logging configuration
 #
+
+
+class MyFilter(logging.Filter):
+    def filter(self, record):
+        record.msg = str(record.msg).replace("'", '"')
+        return True
```

Comment:
Per the docs, filters are used to determine if a given event should or should not be logged. But the docs also say you can use them to do things like this.

Our JSON objects weren't logging correctly because they were using single quotes instead of double quotes.
With this change, we'll now get more powerful papertrail logs like this:

![image](https://github.com/thinknimble/tn-spa-bootstrapper/assets/444070/c8f440ff-becd-4f39-87ce-06b72fb58279)

Reply:
not that it's collapsed by default but theres a little button there to expand/collapse the JSON object.